README.md

```diff
@@ -1,3 +1,26 @@
-# PHP App Store In-App Purchase
+<p align="center">
+    <a href="https://imdhemy.com/laravel-iap-docs/docs/appstore-iap/intro">
+        <img src="logo.png"  alt="Appstore IAP documentation" width="300"/>
+    </a>
+</p>
 
-PHP App store In-App Purchase implementaiton.
+<p align="center">
+<a href="https://packagist.org/packages/imdhemy/appstore-iap"><img src="https://img.shields.io/packagist/dt/imdhemy/appstore-iap" alt="Total Downloads"></a>
+<a href="https://packagist.org/packages/imdhemy/appstore-iap"><img src="https://img.shields.io/packagist/v/imdhemy/appstore-iap" alt="Latest Stable Version"></a>
+<a href="https://packagist.org/packages/imdhemy/appstore-iap"><img src="https://img.shields.io/packagist/l/imdhemy/appstore-iap" alt="License"></a>
+</p>
+
+## About App store IAP
+
+App Store IAP is a php package to handle App store In-app purchase verification and Server Notifications with expressive
```

Comment:
thrown this paragraph into Grammarly if you prefer:
"App Store IAP is a PHP package to handle App Store purchase verification and Server Notifications with expressive and elegant syntax.
This package takes the pain out of development by allowing you to mock receipts and implement their business logic without actual iOS receipts!"

Reply:
Sounds great!